logistic_requisition/model/logistic_requisition.py

```diff
@@ -1150,7 +1150,7 @@ def action_create_po_requisition(self):
 
     @api.multi
     def action_open_po_requisition(self, purch_req=None):
-        if not purch_req:
+        if not purch_req or not isinstance(purch_req, int):
```

Comment:
`action_open_po_requisition` should get a record set as `purch_req` if called from `action_create_po_requisition`.
It get a dict when called from view. I'm not sure what is included in that dict but I think we get the context there.

What you are doing here is breaking this method as purch_req will never be an integer. It is a record set.

I think we should rename this method to `open_po_requisition` with same parameters.
And add a new method

    @api.multi
    open_po_requisition(self, purch_req=None):
        """ Open Purchase order requisition """
        if purch_req is None:
            purch_req = self.po_requisition_id
        ...

    @api.multi
    action_open_po_requisition(self)
        """ Method called from view """
        self.open_po_requisition()